.github/workflows/services.yml

```diff
@@ -0,0 +1,26 @@
+name: Services
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.20'
+
+      - name: Build
+        working-directory: ./packages/services
+        run: go build -v ./...
+
+      - name: Test
+        working-directory: ./packages/services
```

Comment:
if we add this to the `build` and `test` scripts in the `package.json` it will automatically run when running `pnpm build` / `pnpm test` at the root, so we don't need this separate action